cishouseholds/pipeline/survey_responses_version_digital_ETL.py

```diff
@@ -10,4 +12,33 @@ def digital_specific_transformations(df):
     df = df.withColumn("visit_id", F.col("participant_completion_window_id"))
     df = df.withColumn("visit_datetime", F.col("digital_enrolment_invite_datetime"))
     df = transform_survey_responses_generic(df)
+    df = assign_raw_copies(
+        df,
+        [
+            "work_in_additional_paid_employment",
+            "self_isolating",
+            "illness_lasting_over_12_months",
+            "ever_smoked_regularly",
+            "currently_smokes_or_vapes",
+            "hours_a_day_with_someone_else_at_home",
+            "face_covering_work_or_education",
+        ],
+    )
+    mapping_dict = {
+        "Prefer not to say": None,
+        "Don&#39;t Know": None,
```

Comment:
This should match the response we expect to see once the data are correct, rather than the incorrectly encoded versions: "Don't know"